app/models/validations/setup_validations.rb

```diff
@@ -27,6 +36,37 @@ def validate_organisation(record)
 
 private
 
+  def active_collection_start_date
+    if FormHandler.instance.lettings_in_crossover_period?
+      previous_collection_start_date
+    else
+      current_collection_start_date
+    end
+  end
+
+  def validation_error_message
```

Comment:
i'd probably opt for a less generic name. because this apps loves modules, everything will get mixed into one class therefore to prevent collisions a more specific name may be wiser

Reply:
good shout, will do for sales logs too 👍